integrations/firebase_crashlytics_demo_project/README.md

```diff
@@ -1,40 +1,94 @@
-# firebase crashlytics demo project
+# Uploading dSYM artifacts to Firebase Crashlytics
 
+**dSYM** files store the debug symbols for your app. They contain mapping information to decode a stack-trace into a readable format. The purpose of **dSYM** is to replace symbols in the crash logs with the specific methods so it will be readable and helpful for debugging the crash. 
```

Comment:
```suggestion
**dSYM** files store the debug symbols for your app. They contain mapping information to decode a stack trace into a readable format. The purpose of **dSYM** is to replace symbols in the crash logs with the specific methods so it will be readable and helpful for debugging the crash. 
```